src/data/calculate.ts

```diff
@@ -36,7 +39,12 @@ export interface CalculatorData {
   voice: string
 }
 
+export const BUDGET_STANDARD = 10000
```

Comment:
These should be called BUDGET_ONE_PERCENT and BUDGET_TENTH_PERCENT so that it is obvious that changing the numeric value requires changes elsewhere (changing the text).

Reply:
Good call.